daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Error.scala

```diff
@@ -9,33 +9,41 @@ import com.daml.lf.language.Ast
 import com.daml.lf.transaction.{GlobalKey, NodeId}
 import com.daml.lf.value.Value
 
-sealed abstract class Error {
-  def msg: String
+sealed trait Error {
```

Comment:
and here traits are suddenly better than abstract classes? :grin:

Reply:
They are not better :(. I have a conflict with the `RuntimeException` below. 
Will try to put back the abstract class.